modules/govuk_crawler/manifests/init.pp

```diff
@@ -144,7 +144,7 @@
   # This explicitly requires 'base::packages' so that nokogiri will build
   exec { 'gem install govuk_seed_crawler':
     environment => ['RBENV_VERSION=2.6.3'],
-    command     => 'gem install govuk_seed_crawler -v 3.0.0',
+    command     => 'gem install govuk_seed_crawler -v 3.0.0 --bindir /usr/local/bin',
     unless      => 'gem list | grep "govuk_seed_crawler.*3.0.0"',
```

Comment:
Just tested this on integration - it worked 🎉 

```
# no `seed-crawler` in the bin
$ ls -lsa | grep seed-crawler
    4 -rwxr-x---  1 govuk-crawler root     1161 Jul 26 09:49 seed-crawler-wrapper

$ govuk_puppet --test

# now have `seed-crawler` in the bin
$ ls -lsa | grep seed-crawler
    4 -rwxr-xr-x  1 root          root      607 Nov 17 14:48 seed-crawler
    4 -rwxr-x---  1 govuk-crawler root     1161 Jul 26 09:49 seed-crawler-wrapper
```

...however, because of the `unless` line above, it didn't work first time just from running Puppet. I had to manually remove the govuk_seed_crawler gem first (`rbenv shell 2.6.3`, `cd /usr/lib/rbenv/versions/2.6.3/`, `find . -name "*govuk_seed_crawler*"` & delete the files & dirs). So this will need some massaging after the deploy, to get it working initially on Production.

Reply:
I've read [this](https://superuser.com/questions/1103647/effective-difference-between-gem-install-bindir-path-to-folder-and-user) about what `--bindir` does. But I can't see from the [link](https://github.com/alphagov/govuk-puppet/blob/18c30ffd1a8fcd8bf39f48d9ac83dddbcf6a932f/modules/govuk_crawler/templates/seed-crawler-wrapper.erb#L23) you've provided where we previously told puppet to look for the binary before your change. Hope that makes sense!

Reply:
That makes sense 👍 Without specifying `--bindir` manually, the binary was saved in the default location, which was `/usr/lib/rbenv/versions/2.6.3/bin`.